### ISSUE TITLE
How to cancel auto refreshing after table loaded

### ISSUE THREAD

**Issue by litao11046** [wenzhixin/bootstrap-table#5974]
### Description

Is there any way which can cancel the auto refreshing data action when the table loaded.

**Comment by UtechtDustin** [comment]
I'm confused about you're question.
The auto fresh feature will auto refresh the table AFTER the table was loaded.
Which means if you cancel/disable the auto refresh feature after the table was loaded, the table will never refresh the table.

Could you describe your question further ?

**Comment by litao11046** [comment]
I custom a filter panel and the bootstrap table in one web page. When the page loading, the table will also load. But after the table load, it will auto refresh the data. I don't want  this action, cause the filter panel doesn't be set the value before the table loaded.

**Comment by UtechtDustin** [comment]
I still don't know what do you mean, please provide us an example using our [editor](https://live.bootstrap-table.com/https://live.bootstrap-table.com/).

**Comment by litao11046** [comment]
example: https://live.bootstrap-table.com/code/litao11046/9793
this this example, I set a filterPanelForm. I wanna, when the table is ready, do not fetch data automatically. Only when I click the refresh button, then fire the fetching action.

**Comment by UtechtDustin** [comment]
So if i understand correctly, you want custom inputs which will be send to the server for filtering purposes ?
This example should help: https://live.bootstrap-table.com/code/UtechtDustin/9809
You can use either the build in refresh button or a custom one to refresh/filter the table.

If you want that the table is empty on the first request, just check if the submitted data is empty and return an empty dataset.

**Comment by litao11046** [comment]
Yes, I want custom inputs. There is a problem, the filtering field is required for server interface. 

I add the validate action using jquery.validate plugin like the following code. 

```
  function queryParamsfunc(params) {
         var validator = $("#filterPanelForm").validate();
	  if(!validator.form()){
		  return false;
	  }
         
         $("#filterPanelForm").serializeArray().map(function(x){formData[x.name] = x.value;})
         return $.extend(params, formData);
  };
```
However, when the page loaded, queryParamsfunc will perform automatically. Then it will highlight the empty field which is required. This looks not friendly for users.

**Comment by UtechtDustin** [comment]
Then you have the following three options:
- Set the [focusInvalid](https://jqueryvalidation.org/validate/#focusinvalid) option  of the `validate` method to `false`.
- Validate it yourself without focusing
- Check if the input is empty, then don't call the `validate` method.
